pilot/pkg/model/validation.go

```diff
@@ -1713,16 +1721,21 @@ func ValidateQuotaSpecBinding(msg proto.Message) error {
 	return errs
 }
 
-// ValidateAuthenticationPolicy checks that AuthenticationPolicy is well-formed.
-func ValidateAuthenticationPolicy(msg proto.Message) error {
+func validateAuthenticationPolicyImpl(msg proto.Message, clusterScoped bool) error {
 	in, ok := msg.(*authn.Policy)
 	if !ok {
 		return errors.New("cannot cast to AuthenticationPolicy")
 	}
 	var errs error
 
-	for _, target := range in.Targets {
-		errs = appendErrors(errs, validateAuthNPolicyTarget(target))
+	if !clusterScoped {
+		for _, target := range in.Targets {
+			errs = appendErrors(errs, validateAuthNPolicyTarget(target))
+		}
+	} else {
+		if len(in.Targets) > 0 {
```

Comment:
This needs to validate that the object name is `default` for cluster scoped policy. You'll likely need to add a `name` parameter to the validation functions to make this work.

Reply:
Done. Need to change the interface and all other configs though.